app/components/press-release/form.js

```diff
@@ -53,6 +59,19 @@ export default class PressReleaseFormComponent extends Component {
     sources.removeObject(source);
   }
 
+  @action
+  async uploadFile(attachment) {
```

Comment:
Naar analogie met `removeAttachment`, zou ik deze `addAttachment` noemen. Het uploaden van de file zelf is hiervoor al gebeurd.